### PR TITLE
refactor: cast ethereum bridge log recipient address to new version

### PR DESCRIPTION
**Pull request by mkxbl**

### CODE REVIEW

File: offchain-modules/package.json [hunk 1]
```diff
@@ -117,6 +117,6 @@
     }
   },
   "engines": {
-    "node": ">=12.0.0 <15.0.0"
+    "node": ">=14.0.0 <15.0.0"
```
Comment (huwenchao) [inline offchain-modules/package.json:120]:
```suggestion
    "node": "14.x"
```

File: offchain-modules/packages/x/src/handlers/eth.ts [hunk 2]
```diff
@@ -488,3 +489,7 @@ export function parseLockLog(log: Log, parsedLog: ParsedLog): ParsedLockLog {
     asset,
   };
 }
+
+function toCKBAddress2021(address: string): string {
+  return generateAddress(parseAddress(address));
```
Comment (huwenchao) [inline offchain-modules/packages/x/src/handlers/eth.ts:494]: should be refactored after the new release of lumos

Comment (mkxbl) [reply offchain-modules/packages/x/src/handlers/eth.ts:494]: delete short id of lumos config to workaround

File: offchain-modules/package.json (hunk 1 above)
Comment (mkxbl) [reply offchain-modules/package.json]: fixed

**Review by huwenchao — APPROVED** [review]

**Pull request merged**